prow/flagutil/github.go

```diff
@@ -39,6 +40,7 @@ func (o *GitHubOptions) AddFlags(fs *flag.FlagSet) {
 	o.endpoint = NewStrings("https://api.github.com")
 	fs.Var(&o.endpoint, "github-endpoint", "GitHub's API endpoint (may differ for enterprise).")
 	fs.StringVar(&o.TokenPath, "github-token-path", "/etc/github/oauth", "Path to the file containing the GitHub OAuth secret.")
+	fs.StringVar(&o.baseURL, "base-url", "github.com", "the github url that prow will target (may differ for enterprise).")
```

Comment:
Can we infer this from o.endpoint? and/or infer o.endpoint from o.baseURL?

Reply:
Sure